ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml

```diff
@@ -0,0 +1,35 @@
+base_images:
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
+build_root:
+  project_image:
+    context_dir: hack/kubevirt-builder
+    dockerfile_path: Dockerfile
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.2"
+  namespace: ocp
+tests:
+- as: "kubevirt-e2e-test"
+  commands: |
+    set -e
+    set -o pipefail
+
+    export GIMME_GO_VERSION=1.12.8
+    source /etc/profile.d/gimme.sh
+
+    export KUBEVIRT_PROVIDER=external
+    ./hack/generate.sh
```

Comment:
Why don't you call make here ?

Reply:
Because bazel is not available.

Reply:
BTW: now bazel is available as we can use the kubevirt-builder, but still docker is not available, so we can't use anything that uses `hack/dockerized`

Reply:
We can directly use the builder image as a base, so the good thing is that we don't need to extra start the builder container to use its content. :+1: